### ISSUE TITLE
target grid num is 14

### ISSUE THREAD

**Issue by Songyima** [xiongzihua/pytorch-YOLO-v1#11]
I found that target grid num is 14， and error while training. So I change it to 7.
what's more, the loss never decrease when it reach 4.xx
I use it on people detect, mAP is only 0.08

**Comment by abeardear** [comment]
448 input size, output size will be 14
